akka-http-tests/src/test/scala/akka/http/scaladsl/server/directives/FileAndResourceDirectivesSpec.scala

```diff
@@ -27,7 +27,8 @@ class FileAndResourceDirectivesSpec extends RoutingSpec with Inspectors with Ins
   // operations touch files, can be randomly hit by slowness
   implicit val routeTestTimeout = RouteTestTimeout(3.seconds)
 
-  val testRoot = new File(getClass.getClassLoader.getResource("").toURI)
+  val testRoot = new File("akka-http-tests/src/test/resources")
+  require(testRoot.exists(), s"testRoot was not found at ${testRoot.getAbsolutePath}")
```

Comment:
Because, otherwise, files will be served from the `target` directory where sbt will have copied all files over, resolving symlinks in the process.